erpnext/accounts/report/accounts_receivable/accounts_receivable.py

```diff
@@ -128,6 +128,7 @@ def init_voucher_balance(self):
 					credit_note_in_account_currency=0.0,
 					outstanding_in_account_currency=0.0,
 				)
+			self.get_invoices(ple)
```

Comment:
Just a note on naming, a method called `get_SOMETHING` should not be mutating state.